dpnp/backend/kernels/dpnp_krnl_bitwise.cpp

```diff
@@ -226,6 +226,14 @@ static void func_map_init_bitwise_1arg_1type(func_map_t& fmap)
             };                                                                                                         \
             event = q.submit(kernel_func);                                                                             \
         }                                                                                                              \
+        input1_ptr.depends_on(event);                                                                                  \
+        input1_shape_ptr.depends_on(event);                                                                            \
```

Comment:
Looks both input1_shape_ptr and input2_shape_ptr aren't used in the kernel. Should the dependency on event be created for them anyway?